jwst/datamodels/wcs_ref_models.py

```diff
@@ -166,6 +167,9 @@ class SpecwcsModel(_SimpleModel):
     schema_url = "http://stsci.edu/schemas/jwst_datamodel/specwcs.schema"
     reftype = "specwcs"
 
+    def on_save(self, path=None):
+        self.meta.reftype = self.reftype
+
```

Comment:
This is not needed now as `_SimpleModel.on_save` is already defined, and `SpecwcsModel` inherits it.

Reply:
Removed, and confirmed that the files still look ok.